engine/src/test/scala/cromwell/engine/io/gcs/GcsBatchCommandContextSpec.scala

```diff
@@ -16,59 +18,91 @@ import scala.util.{Failure, Success}
 class GcsBatchCommandContextSpec extends AnyFlatSpec with CromwellTimeoutSpec with Matchers with Eventually with BeforeAndAfter  {
   behavior of "GcsBatchCommandContext"
 
-  var commandContext: GcsBatchCommandContext[Unit, Unit] = _
+  var exceptionSpewingCommandContext: GcsBatchCommandContext[Unit, Unit] = _
+
+  var errorReturningCommandContext: GcsBatchCommandContext[Unit, Unit] = _
 
   before {
-    commandContext = GcsBatchCommandContext[Unit, Unit](
+    exceptionSpewingCommandContext = GcsBatchCommandContext[Unit, Unit](
```

Comment:
TOL these might be better as `def`s rather than `vars`-with-`before`-setters since only one or the other is used per test.

TBH I'm not 100% sure why I didn't make them `def`s originally... 🤷‍♂️

Reply:
caught in the middle of a self-seagulling... 🐦

Reply:
actually that turns out to be problematic since a `def` gives every invocation in each test method a different context and that is not good for sanity.